pkg/cloud/services/wait/wait.go

```diff
@@ -48,7 +48,8 @@ func NewBackoff() wait.Backoff {
 
 // WaitForWithRetryable repeats a condition check with exponential backoff.
 func WaitForWithRetryable(backoff wait.Backoff, condition wait.ConditionFunc, retryableErrors ...string) error { //nolint
-	return wait.ExponentialBackoff(backoff, func() (bool, error) {
+	var actualErr error
```

Comment:
Maybe we should call this something like `errToReturn`?

Reply:
addressed at #1221